src/main/webapp/go.js

```diff
@@ -184,28 +225,27 @@ function handleDestinationAnswer() {
  * Disable lint check because startHunt() is called from go.html.
  */
 function startHunt() { //eslint-disable-line
-  destIndex = 0;
+  hunt.start();
   sendIndexToServlet(0);
-  updateToCurrentState(0);
+  updateToCurrentState();
 }
 
 /**
  * Updates the hunt to the current destination that the user is on.
- * @param {int} index The index of the destination that the user needs
- * to find.
  */
-function updateToCurrentState(index) {
-  if (index <= -1) { // The user has not yet pressed the start button.
+function updateToCurrentState() {
+  if (hunt.getDestIndex() <= -1) { // User has not pressed the start button.
```

Comment:
You added a helper for this!
```suggestion
  if (hunt. hasNotStarted()) { // User has not pressed the start button.
```